app/helpers/reference_helper.rb

```diff
@@ -0,0 +1,38 @@
+module ReferenceHelper
+
+  def after_desired_date?
+    Time.zone.today > Date.parse('2015-12-31')
```

Comment:
What you say is right, testability. So we have two approaches:

* use ENV var as you suggest
* get Filippos to fast forward the time on the demo server by 1 month and it'll be testable (then reset the server's time back to the correct time)

With both approaches we'll require Filippos' time. With the first approach, it'll be setting the ENV var in the **fr-deploy** repo plus me modifying the code. In the second, as mentioned above.

The question is, what trade off do we want to make: doing something cleanly that'll take a bit more time or going with what we've got that'll take less time?